api-tests/docker-compose.yml

```diff
@@ -12,10 +12,10 @@ services:
       - PMM_DEBUG=1
       - PERCONA_TEST_CHECKS_INTERVAL=10s
       # for local development
-      # - PERCONA_TEST_CHECKS_FILE=/srv/checks/custom-checks.yml
+      # - PMM_ADVISORS_CHECKS_FILE=/srv/checks/custom-checks.yml
       # for check-dev
-      - PERCONA_TEST_SAAS_HOST=check-dev.percona.com
-      - PERCONA_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
+      - PMM_TEST_SAAS_HOST=check-dev.percona.com
+      - PMM_TEST_CHECKS_PUBLIC_KEY=RWTg+ZmCCjt7O8eWeAmTLAqW+1ozUbpRSKSwNTmO+exlS5KEIPYWuYdX
```

Comment:
can we have the same env variables as for https://github.com/percona/pmm/pull/2857/files#diff-8c3f82f0b002430c7b079361d8dbc0662a32467f16bc9d20b5796745488abdf8R98-R99